doc/common/README.md

```diff
@@ -758,4 +758,68 @@ a. Каждое новое предложение по стилю кода до
 b. Правило добавляется с пометкой `\[Не автоматизировано\]` в заголовке правила.
 Например: `3.2.2. \[Не автоматизировано\] Правила именования для обработчков`.
 
-c. Должна быть создана задача на автоматизацию правила.
\ No newline at end of file
+c. Должна быть создана задача на автоматизацию правила.
+
+## 9. Правила именования
+
+### 9.1. \[Не автоматизировано\] Общие правила для всех идентификаторов
+
+a. Используются только буквы латинского алфавита `[a-zA-Z]`, цифры `[0-9]`. 
```

Comment:
Do we really need numbers here?

Reply:
Good question! Thanky you! It should be really rare case, but it can be. For instance, we will use name of specification as variable name or enum key:

```
enum SpecType {
   JSR301,
   ...
}
```